tests/cli/test_suite.py

```diff
@@ -24,12 +24,76 @@
 )
 from great_expectations.util import deep_filter_properties_iterable, lint_code
 from tests.cli.utils import assert_no_logging_messages_or_tracebacks
+from tests.render.renderer.v3.test_suite_profile_notebook_renderer import (
+    EXPECTED_EXPECTATION_CONFIGURATIONS_USER_CONFIGURABLE_PROFILER,
+)
 from tests.render.test_util import (
     find_code_in_notebook,
     load_notebook_from_path,
     run_notebook,
 )
 
+PROFILER_CODE_CELL_USER_CONFIGURABLE_PROFILER: str = """\
+profiler = UserConfigurableProfiler(
+    profile_dataset=validator,
+    excluded_expectations=None,
+    ignored_columns=exclude_column_names,
+    not_null_only=False,
+    primary_or_compound_key=None,
+    semantic_types_dict=None,
+    table_expectations_only=False,
+    value_set_threshold="MANY",
+)
+suite = profiler.build_suite()
+validator.expectation_suite = suite
+"""
+PROFILER_CODE_CELL_ONBOARDING_DATA_ASSISTANT: str = """\
+data_assistant_result: DataAssistantResult = context.assistants.onboarding.run(
+    batch_request=batch_request,
+    # include_column_names=include_column_names,
+    exclude_column_names=exclude_column_names,
+    # include_column_name_suffixes=include_column_name_suffixes,
+    # exclude_column_name_suffixes=exclude_column_name_suffixes,
+    # semantic_type_filter_module_name=semantic_type_filter_module_name,
+    # semantic_type_filter_class_name=semantic_type_filter_class_name,
+    # include_semantic_types=include_semantic_types,
+    # exclude_semantic_types=exclude_semantic_types,
+    # allowed_semantic_types_passthrough=allowed_semantic_types_passthrough,
+    cardinality_limit_mode="rel_100",  # case-insenstive (see documentaiton for other options)
```

Comment:
Something smells wrong about how this is the only config not commented out.

Reply:
@NathanFarmer Fixing in #5310 -- thanks!